### PR TITLE
Remove `TODO` comment for closed issue

### PR DESCRIPTION
**Pull request by adetunjii**
# Description
This fix removes `TODO` for closed out issues.

## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [ ] I added/updated comments and checked rendering.
* [ ] I made spot refactorings.
* [ ] I updated user documentation.
* [ ] I ran `task all`, and it passed.
* [ ] I ensured that PR title is good enough for the changelog.
* [ ] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Labels, Project and project's Sprint fields.
* [ ] I marked all done items in this checklist.

### CODE REVIEW

**Review by chilagrow — DISMISSED** [review]
Thanks for the clean up :hugs:

File: internal/wire/msg_body.go [hunk 1]
```diff
@@ -44,6 +44,7 @@ var ErrZeroRead = errors.New("zero bytes read")
 //
 // Error is (possibly wrapped) ErrZeroRead if zero bytes was read.
 func ReadMessage(r *bufio.Reader) (*MsgHeader, MsgBody, error) {
+
```
Comment (AlekSi) [inline internal/wire/msg_body.go:47]: Linter complains about that empty line

**Comment by AlekSi** [comment]
@adetunjii please click a button in this comments: https://github.com/FerretDB/FerretDB/pull/2573#issuecomment-1537115982

File: internal/wire/msg_body.go (hunk 1 above)
Comment (adetunjii) [reply internal/wire/msg_body.go]: @AlekSi Fixed

**Review by AlekSi — APPROVED** [review]

**Review by chilagrow — APPROVED** [review]

**Pull request merged**